### DISCUSSION TITLE
WARNING  Will not infuse 440017b9be0144c401.....d202fe because its reward chai

### DISCUSSION THREAD

**Discussion by elmans1961** [Chia-Network/chia-blockchain#7964] [Support]
What's wrong with my log files?The following 2 messages I see only:
1.timelord chia.timelord.timelord   : WARNING  Will not infuse 440017b9be0144c40185af636c80f....71816d202fe because its reward chain challenge is not in the chain
2.farmer chia.farmer.farmer         : ERROR    Could not find authentication sk for pk: 8f0518494357d6e0....0d0224cdb5884ae3e5f4b7b2c8703ec6f8888388b8e79
I upgreded from version 1.2.2 to 1.2.3.It began.I deleted completly chia fom my PC including folders C:\Users\admin\.chia ,C:\Users\admin\AppData\Local\chia-blockchain and C:\Users\admin\AppData\Roaming\Chia Blockchain and installed chia from the scratch bur it does not help.What shall I do?

**Comment by keliew** [comment]
Don't think the Timelord is compatible with Windows...
